### PR TITLE
EQAditu.AdvancedCombatTracker version 3.5.0.273

### PR DESCRIPTION
**Pull request by andrewbabbittdev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19346)

### CODE REVIEW

File: manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: "EQAditu.AdvancedCombatTracker"
+PackageVersion: "3.5.0.273"
+PackageLocale: "en-US"
+Publisher: "EQAditu"
```
Comment (OfficialEsco) [inline manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml:6]: Publisher is blank in ARP (Programs and Features) could you contact the dev about this?

File: manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: "EQAditu.AdvancedCombatTracker"
+PackageVersion: "3.5.0.273"
```
Comment (OfficialEsco) [inline manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml:4]: Version is blank in ARP (Programs and Features) could you contact the dev about this?

**Review by OfficialEsco — APPROVED** [review]
This information is required for `winget list` and `winget upgrade` to work.

File: manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml (hunk 1 above)
Comment (AndrewBabbitt97) [reply manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml:6]: Tracking in EQAditu/AdvancedCombatTracker#33

File: manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml (hunk 2 above)
Comment (AndrewBabbitt97) [reply manifests/e/EQAditu/AdvancedCombatTracker/3.5.0.273/EQAditu.AdvancedCombatTracker.locale.en-US.yaml:4]: Tracking in EQAditu/AdvancedCombatTracker#33

**Pull request merged**